java/src/jmri/jmrix/can/swing/CanPanel.java

```diff
@@ -56,4 +56,8 @@ public String prependConnToString(String initial) {
         return initial;
     }
 
+    @Override
+    public void dispose() {
+        super.dispose();
+    }
```

Comment:
Why is this needed? If the method `dispose()` does not exists in this class, the parent method will be called.